lib/rspec-puppet/setup.rb

```diff
@@ -55,7 +55,11 @@ def self.get_module_name
     end
 
     def self.get_module_name_from_file(file)
-      p = Puppet::Parser::Lexer.new
+      if Puppet.version.to_f >= 4.0
+        p = Puppet::Pops::Parser::Lexer2.new
```

Comment:
I can confirm that the method is flawed, and it returns the wrong result for the actual puppet code I'm using here as input.  But, as noted, it was broken against Puppet 3 as well.  As a quick and dirty hack, though, I'm sure this hack lets lots of people quickly set up their puppet-rspec.  Perhaps I should add a "FIXME" note per @hlindberg's comments?